request_type.go

```diff
@@ -0,0 +1,203 @@
+package hedera
+
+import (
+	"fmt"
+)
+
+type RequestType uint32
+
+
+
+const (
+	RequestTypeNONE           RequestType = 0 // UNSPECIFIED - Need to keep first value as unspecified because first element is ignored and not parsed (0 is ignored by parser)
```

Comment:
Unlike the protobufs, lets be consistent. All enum variants should be `PascalCase` so `RequestTypeNone`.
Also, can you move all the comments above each enum instead of after it. I don't think we use the comments after code anywhere in the code base.